spec/client_spec.rb

```diff
@@ -20,6 +20,7 @@
                             liqui
                             gemini
                             hitbtc
+                            bittrex
```

Comment:
@Swingcloud actually the real way to test this is to arrange the order yourself manually without calling `sort` because you're calling `sort` on the actual method as well. If `sort` isn't behaving the way you want, you wouldn't have known because the test will still pass.